src/Http/Controllers/ChannelController.php

```diff
@@ -74,6 +75,13 @@ public function show(Channel $channel)
         return view('vanilo::channel.show', $this->processViewData(__METHOD__, [
             'channel' => $channel,
             'products' => (new ProductSearch())->withinChannel($channel)->withInactiveProducts()->getResults(),
+            'country' => CountryProxy::where('id', $channel->configuration()['country_id'])->first(),
+            'orderCount' => DB::table('orders')->where('channel_id', $channel->id)->count(),
+            'ordersPerCurrency' => DB::table('orders')
+                ->select('currency', DB::raw('COUNT(*) as order_count'))
```

Comment:
Use SUM, not COUNT

I guess, you want to know how many **dollars** you have in your pocket and not how many **banknotes** :wink:

Reply:
I mean we want to know the SUM of order totals per currency not their count